app/jobs/docusign/void_childcare_envelope_job.rb

```diff
@@ -0,0 +1,13 @@
+class Docusign::VoidChildcareEnvelopeJob < ApplicationJob
+  queue_as :default
+
+  # Stop from retrying if envelope is already voided, declined or completed
+  rescue_from(Childcare::VoidDocusignEnvelope::VoidEnvelopeError) do |exception|
+    Rollbar.error(exception)
+    Rails.logger.error "[#{self.class.name}] Something went wrong with you job: #{exception}"
```

Comment:
The Rollbar + Rails.logger pattern is becoming common, is it worth extracting somewhere? Maybe not, just a thought.